vocabularies/security-classifications.ttl

```diff
@@ -37,6 +36,14 @@ qgiscf:official a skos:Concept ;
  rdfs:label "OFFICIAL"@en ;
  skos:prefLabel "OFFICIAL"@en  .
 
+qgiscf:official-public a skos:Concept ; 
+ skos:broader qgiscf:official ;
+ skos:definition "As per OFFICIAL with the Dissemination Limiting Marking (DLM) applied to indicates the information is publically available and may be accessed by or released to the public."@en ;
+ skos:inScheme <http://linked.data.gov.au/def/qg-security-classifications> ;
+ skos:notation "off" ;
```

Comment:
```suggestion
 skos:notation "ofp" ;
```
notations must be unique for the same reason as labels. Just a suggested notation, anything unique will be fine.